server/routes/induction/update/highestLevelOfEducationUpdateController.ts

```diff
@@ -65,11 +65,13 @@ export default class HighestLevelOfEducationUpdateController extends HighestLeve
         return next(createError(500, `Error updating Induction for prisoner ${prisonNumber}. Error: ${e}`))
       }
     } else {
-      logger.debug(
-        'The new Highest Level of Education requires asking further questions about educational qualifications',
+      logger.debug('New Highest Level of Education requires asking further questions about educational qualifications')
+      req.session.inductionDto = updatedInductionDtoWithHighestLevelOfEducation(
```

Comment:
This updates the highest level of education within the Induction DTO in the session (which I think is all we need to do, before redirecting to the "Qualification Level" page).

Reply:
> which I think is all we need to do

Until we need the `pageFlowQueue` to encapsulate the mini-flow as discussed just now ! 😉 

This is fine for now, but I think we'll be using a `pageFlowQueue` in this branch of code pretty soon! 🤣